test/unit/parameters.test.js

```diff
@@ -0,0 +1,340 @@
+/* global describe, it */
+
+import assert from 'assert';
+import PubNub from '../../src/node/index';
+
+let pubnub = new PubNub({ subscribeKey: 'mySubKey', publishKey: 'myPublishKey', uuid: 'myUUID', authKey: 'myAuthKey' });
```

Comment:
@ManuelFernando can we move those tests into the specific test files for the endpoints?